### ISSUE TITLE
Error 400 on follow request

### ISSUE THREAD

**Issue by sahandmax** [ohld/igbot#466]
hi 
on follow i got below message

2018-03-15 09:25:34,223 - ERROR - Request return 400 error!
2018-03-15 09:25:34,247 - INFO - Instagram error message: feedback_required

**Comment by basnijholt** [comment]
Could you please provide more information and the code that lead you to get this error?

**Comment by lasododo** [comment]
I get the same error after running message_users.py , but I got it after it sent a message to 140 people... the code is competlu same, there is just changed message

**Comment by basnijholt** [comment]
@lasododo, like my comment above: Could you please provide more information and the code that lead you to get this error?

And besides that, the complete error.

**Comment by lasododo** [comment]
```
C:\Users\PC\PycharmProjects\untitled\venv\Scripts\python.exe C:/Users/PC/PycharmProjects/instagram-workinfDMMaybe/instabot-master/examples/message_users.py
Which type of delivery method? (Type number)
0: Messages From CSV File.
1: Group Message All Users From List.
2: Message Each User From List.
3: Message Each Your Follower.
4: Message LatestMediaLikers Of A Page
3
Which account do you want to use? (Type number)
2018-04-23 15:37:58,264 - INFO - Instabot Started
1: ************
0: add another account.
-1: delete all accounts.
1
2018-04-23 15:38:00,881 - INFO - Logged-in successfully as '**************'!
  0%|          | 0/1858 [00:00<?, ?it/s]2018-04-23 15:38:07,351 - ERROR - Request returns 400 error!
2018-04-23 15:38:07,352 - INFO - Instagram's error message: feedback_required
2018-04-23 15:38:07,353 - INFO - Message to ['5373273860'] wasn't sent
 34%|███▍      | 640/1858 [00:20<00:38, 32.00it/s] 2018-04-23 15:38:56,081 - ERROR - Request returns 400 error!
2018-04-23 15:38:56,082 - INFO - Instagram's error message: feedback_required
2018-04-23 15:38:56,082 - INFO - Message to ['7099486693'] wasn't sent
 34%|███▍      | 641/1858 [00:52<01:38, 12.30it/s]
```

This is the whole error message, ( I have set it up now to jump into 640th follower, just in case that its coused by someone.... but if I choose an option "Message Each User From List." it will send all messages, just for followers its causing this error, isnt there somekind of protection agains abusing this bot ? like after 140 messages to calm down for 24 hours or something ? + if you want to I can pack up the whole file and send it to you

**Comment by lasododo** [comment]
[instabot-master.zip](https://github.com/instagrambot/instabot/files/1938438/instabot-master.zip)
here is the project in case you woud like to see it

**Comment by basnijholt** [comment]
See https://github.com/huttarichard/instagram-private-api/issues/330 and https://github.com/mgp25/Instagram-API/issues/1577

It seems like your account is being blocked because of over usage.

**Comment by lasododo** [comment]
okey thank you, just a side question, if its blocked how can I still send message to my personal account from that acc then ?

**Comment by lubislutfi** [comment]
I modified ultimate_schedule script & use it for following user followers. In short, I set the follow delay between 3 and 5 minutes.
Everything is ok, even for many days when I  run only one script(one account). 
But the scripts return 400 error when I copy  that script in multiple folder for running multiple different account simultaneously. I've changed the device setting. One device for one account.
